data/remnant/remnant.txt

```diff
@@ -287,8 +287,8 @@ shipyard "Remnant"
 	"Petrel"
 	"Tern"
 
-shipyard "Remnant Gascraft"
-	"Gascraft"
+shipyard "Remnant Puffin"
```

Comment:
Without a `shipyard "Remnant Gascraft"`, all old saves will cause an error. Simply changing the "Gascraft" in this shipyard to the "Puffin" will mean that you need to make zero changes for compatibility events.

Reply:
If I put a "remove shipyard "Gascraft" " in the event, will it cause a problem when there is no such shipyard to remove?

Reply:
Sidestepped the issue and resolved it by adding a `clear "event: remnant: gascraft"` line to the compatibility mission. This removes the record of the old event entirely.